components/ruby/lib/chef-licensing/string_refinements.rb

```diff
@@ -0,0 +1,24 @@
+module ChefLicensing
+  # String refinements to replace ActiveSupport methods
```

Comment:
I think it's worth doing this ourselves here since it's so simple, but we COULD use linguistics module if we wanted